File: nodejs/src/v5/delivery_metrics.js
/**
 * This module provides a compatible interface for an endpoint
 * that exists in v3 but not v5.
 */

// Use this class to get and to print all of the available
// advertising delivery metrics.
export class DeliveryMetrics {
  // No constructor is required, because the parent constructor
  // should be called automagically. See:
  //   https://eslint.org/docs/rules/no-useless-constructor

  // The full list of all available delivery metrics is available in v3 but not v5.
  async get() {
    throw new Error('Metric definitions are not available in API version v5.');
  }
}

File: python/src/v5/delivery_metrics.py
class DeliveryMetrics:
    """
    This class provides a compatible interface for an endpoint
    that exists in v3 but not v5.
    """

    def __init__(self, api_config, access_token):
        pass

    def get(self):
        """
        The full list of all available delivery metrics is available in v3 but not v5.
        """
        raise RuntimeError("Metric definitions are not available in API version v5.")
